src/panels/MissionsPanel/MissionsPanel.tsx

```diff
@@ -23,5 +37,22 @@ export function MissionsPanel() {
     );
   }
 
-  return <MissionContent missionOverview={mission} />;
+  return (
+    <Box>
+      {data.missions.length > 1 && (
+        <Select
+          label={'Selected mission'}
+          placeholder={'Select a mission'}
+          data={data.missions.map((mission) => {
+            return { value: mission.identifier, label: mission.name };
+          })}
+          value={selectedMissionIdentifier}
+          onChange={onMissionSelected}
+          my={'xs'}
+          allowDeselect={false}
+        />
+      )}
+      <MissionContent missionOverview={mission!} />
```

Comment:
```suggestion
      <MissionContent missionOverview={mission} />
```

and move the check to the if statement above? See other comment